### PR TITLE
add readme

### PR DESCRIPTION
**Pull request by Keyshal-211**
## Description of changes
Please provide a short description of the change or if the change is sufficiently described in the GitHub issue, provide a link to the issue.

  
## How has this been tested?

Please describe the test strategy you followed. If not tested, please explain why.

- [ ] automated unit test
- [ ] automated integration test
- [ ] manual, exploratory test

In case of manual test, please document the test well including a set of user instructions and prerequisites. Each including an action, it's result, and where appropriate a screenshot.
    
## Checklist before requesting a review

- [X] My code is easy to understand
- [x] My code follows the [coding guidelines](https://github.com/IT-REX-Platform/wiki/blob/main/dev-manuals/backend/coding-guidelines.md) of this project
- [ ] My code fulfills all acceptance criteria
- [ ] The test coverage (line and branch) is reasonably high, especially on Service classes and other classes with complex logic
- [x] I have made corresponding changes to the documentation or
  the [wiki/adr](https://github.com/IT-REX-Platform/wiki/tree/main/adr)
- [ ] I made no breaking changes in the database schema or if so, I will perform a database migration

## Checklist for reviewer

- The code is easy to understand
- The code follows
  the [coding guidelines](https://github.com/IT-REX-Platform/wiki/blob/main/dev-manuals/backend/coding-guidelines.md) of
  this project
- The code is tested or if not, the reason is documented or discussed
- The added and existing tests reasonably cover the code change
- The code has no breaking changes in the database schema or if so, the assignee is aware of it and
  will perform a database migration

### CODE REVIEW

**Review by PaulBredl — CHANGES_REQUESTED** [review]
The description is correct in its content, but still idk, your description sounds more like a markting text generated with chat gpt than what I would expect in a readme file. The description should be a neutral, objective description of what this service does.

File: README.md [hunk 1]
```diff
@@ -1,123 +1,25 @@
...
+1. **Content Upload and Management:** It provides a platform for educators to upload and manage multimedia content, including PDF documents, MP4 videos, PowerPoint presentations (PPTX), and more.
 
-Detailed description of the packages:
+2. **MinIO Object Storage Integration:** To ensure efficient storage and accessibility, the service utilizes the MinIO object storage system. Different types of media are stored in separate buckets within MinIO, allowing for structured storage.
 
-### Root package
+3. **GraphQL Endpoint:** Users can interact with the service through a GraphQL endpoint, enabling the retrieval of URLs for uploading and downloading media files.
 
-This should be named after the microservice itself. This is the root package for the microservice. It contains the `Application.java` file (or of similar name), which is the entry point for the microservice. Usually, this is the only class in this package.
+## Purpose
```
Comment (PaulBredl) [inline README.md:11]: Here I would also remove, its almost a repetition of before

Comment (Keyshal-211) [reply README.md]: done @PaulBredl

**Review by PaulBredl — APPROVED** [review]
just resolve merge conflicts

**Pull request merged**